addons/website/views/assets.xml

```diff
@@ -1,112 +1,6 @@
-    </xpath>
-</template>
-
-<template id="_assets_frontend_helpers_user_custom" inherit_id="web._assets_frontend_helpers" priority="99">
-    <xpath expr="//link" position="before">
-        <!-- Custom empty file for user custom bootstrap override -->
-        <link rel="stylesheet" type="text/scss" href="/website/static/src/scss/user_custom_bootstrap_overridden.scss"/>
-    </xpath>
-</template>
-
-<template id="assets_frontend" inherit_id="portal.assets_frontend" name="Website Assets">
-    <xpath expr="//link[last()]" position="after">
-        <link rel="stylesheet" type="text/scss" href="/website/static/src/scss/website.scss"/>
-        <link rel="stylesheet" type="text/scss" href="/website/static/src/scss/website.ui.scss"/>
-    </xpath>
-    <xpath expr="//script[@src='/web/static/src/js/public/public_root_instance.js']" position="replace">
-        <script type="text/javascript" src="/website/static/src/js/content/website_root_instance.js"/>
-    </xpath>
-    <xpath expr="//script[last()]" position="after">
-        <script type="text/javascript" src="/website/static/src/js/set_view_track.js"/>
-        <script type="text/javascript" src="/website/static/src/js/utils.js"/>
-
-        <script type="text/javascript" src="/website/static/src/js/content/website_root.js"/>
-        <script type="text/javascript" src="/website/static/src/js/content/compatibility.js"/>
-        <script type="text/javascript" src="/website/static/src/js/content/menu.js"/>
-        <script type="text/javascript" src="/website/static/src/js/content/snippets.animation.js"/>
-
-        <script type="text/javascript" src="/website/static/src/js/menu/navbar.js"/>
-
-        <script type="text/javascript" src="/website/static/src/js/show_password.js"/>
-        <script type="text/javascript" src="/website/static/src/js/post_link.js"/>
-        <!-- Custom empty file for user javascript -->
-        <script type="text/javascript" src="/website/static/src/js/user_custom_javascript.js"/>
-    </xpath>
-</template>
-
-<template id="assets_frontend_user_custom" inherit_id="web.assets_frontend" name="Website Assets user custom" priority="99">
-    <xpath expr="//link[last()]" position="after">
-        <!-- Custom empty file for user custom rules -->
-        <link rel="stylesheet" type="text/scss" href="/website/static/src/scss/user_custom_rules.scss"/>
-    </xpath>
-</template>
-
-<template id="assets_frontend_compatibility_for_12_0" inherit_id="website.assets_frontend" active="False">
-    <xpath expr="//link[last()]" position="after">
-        <link rel="stylesheet" type="text/scss" href="/website/static/src/scss/compatibility/bs3_for_12_0.scss"/>
-    </xpath>
-</template>
-
 <template id="website.compiled_assets_wysiwyg" name="Website Editor Assets (used in website editor)" groups="website.group_website_publisher">
     <t t-call-assets="website.assets_wysiwyg"/>
 </template>
```

Comment:
Not sure why but your algo didn't pick the templates bellow (`website.assets_wysiwyg` and `website.assets_editor`). Even though the former is a bit tricky since it does not extend `web_editor.assets_wysiwyg` and since we do not namespace bundle names anymore it should be manually renamed before converting.